.gitignore

```diff
@@ -31,4 +31,6 @@
 
 *.log*
 
-triplestore/
\ No newline at end of file
+triplestore/
+
+application-dev.yml
```

Comment:
hi @nymbyl how will a developer know what to put into the application-dev.yml? maybe we could be something in the README?  @wellingWilliam any thoughts on this?

Reply:
I put that in there because my kludged together "scholars-discovery-setup" project seemed to keep copying it in (even though I was trying to just link it up in the docker compose file).  So I wanted to avoid accidentally adding it to the repo.  I would consider the README setup instructions to happen later.  But the "scholars-discovery-setup" does actually have the file itself.